src/community/resources/index.md.njk

```diff
@@ -0,0 +1,103 @@
+---
+title: Resources
+description: Useful resources and community projects
+section: Community
+layout: layout-pane.njk
+order: 1
+status: Experimental
+---
+
+A curated list of community developed resources that can help you use the GOV.UK Design System in your chosen design tool or development environment. 
+
+The Design System team is not responsible for these resources and is unable to provide support for them. If you have support or feature requests please contact the owner.
+
+
+## Create flow diagrams
+
+[GOV.UK Design System Flow Diagrams](https://github.com/dashouse/govuk-design-system-flow-diagrams)⁠ -  
+A Sketch file for creating flow diagrams of GOV.UK services
+
+[GOV Flow](https://github.com/charlesrt/gov-flow)⁠ -  
+A Sketch file for creating flow diagrams of GOV.UK services
+
+[GOV Flow for Google Drawings](https://github.com/charlesrt/gov-flow)⁠ -  
+Flow diagram templates for Google drawings, based on GOV Flow
+
+https://www.beatnic.co.uk/2019/10/04/google-drawing-template-gov-flow-kit/
+
+[GOV.UK Userflow Icons](https://github.com/alirawashdeh/govuk-userflow-icons) -  
+SVG images for creating flow diagrams of GOV.UK services
+
+## Create prototypes and wireframes
+
+[govuk-design-system-balsamiq](https://github.com/enoranidi/govuk-design-system-balsamiq)⁠ -  
+Balsamiq wireframes based on the GOV.UK Design System
+
+[Sketch Wireframing Kit](https://github.com/abbott567/sketch_wireframing_kit)⁠ -  
+Sketch wireframes based on the GOV.UK Design System
+
+## Build front ends
+
+### C#
+
+[GOV.UK Design System in C#/ASP.Net Core](https://github.com/cabinetoffice/govuk-design-system-dotnet)⁠ -  
+A collection of GOV.UK Frontend components built in C# / ASP.Net Core
+
+
+### Node.js
+
+[Hapi GOV.UK examples](https://github.com/DEFRA/hapi-govuk-examples)⁠ -  
```

Comment:
I think this one could just be a link with the description built in. Something like:

Guidance on [building a hapi service using GOV.UK Frontend](https://github.com/DEFRA/hapi-govuk-examples)